src/explore-education-statistics-admin/src/pages/release/data/components/DataFileUploadForm.tsx

```diff
@@ -37,6 +39,10 @@ function baseErrorMappings<FormValues extends DataFileUploadFormValues>(
           DataZipFileDoesNotContainCsvFiles:
             'ZIP file does not contain any CSV files',
           DataFilenameNotUnique: 'Choose a unique ZIP data file name',
+          DataZipFilenameTooLong: `Maximum ZIP data filename cannot exceed ${MAX_FILENAME_SIZE} characters`,
+          DataFilenameTooLong: `Maximum data filename cannot exceed ${MAX_FILENAME_SIZE} characters`,
+          MetaFilenameTooLong: `Maximum metadata filename cannot exceed ${MAX_FILENAME_SIZE} characters`,
+          DataZipContentFilenamesTooLong: `Maximum data and metadata filenames cannot exceed ${MAX_FILENAME_SIZE} characters`,
```

Comment:
You collect all errors in one `ValidateFilenameLengths` method but then create `DataZipCOntentFilenamesTooLong` for the other. Is there a reason why, or maybe `DataZipContentFilenamesTooLong` can be removed?

Reply:
The zip file field can only handle a single error message response (any other items returned in a list are ignored), but the error may relate to both files contained within the archive (data and metadata).

`DataZipContentFilenamesTooLong` combines validation for both content files and ensures the user only has to resubmit the form once if both filenames are too long (as opposed to being told the data filename is too long, submitting, then being told the meta filename is too long).